CMakeLists.txt

```diff
@@ -12,6 +12,14 @@ file(GLOB_RECURSE HTT_UTILITIES_SRC "${CMSSW_BASE_SRC}/HTT-utilities/*.cc")
 add_library(HTT-utilities STATIC ${HTT_UTILITIES_SRC})
 set_source_files_properties(${HTT_UTILITIES_SRC} PROPERTIES COMPILE_FLAGS "-w")
 
+file(GLOB_RECURSE TauAnalysisTool_SRC "${CMSSW_BASE_SRC}/TauAnalysisTools/*.cc")
+add_library(TauAnalysisTools STATIC ${TauAnalysisTool_SRC})
+set_source_files_properties(${TauAnalysisTool_SRC} PROPERTIES COMPILE_FLAGS "-w")
```

Comment:
in theory, `set_source_files_properties` is not needed. The code should compile without warnings. If not, let me know.